src/ol/interaction/Draw.js

```diff
@@ -1666,12 +1668,13 @@ class Draw extends PointerInteraction {
    * Stop drawing and add the sketch feature to the target layer.
    * The {@link module:ol/interaction/Draw~DrawEventType.DRAWEND} event is
    * dispatched before inserting the feature.
+   * @return {Feature<import("../geom/SimpleGeometry.js").default>|null} The drawn feature.
```

Comment:
Hardly seems worth it, but we could add a note to the upgrade notes saying that `finishDrawing` now returns a feature.